.ci/azure-pipelines/linux.yml

```diff
@@ -4,12 +4,11 @@ jobs:
   pool:
     vmImage: ubuntu-16.04
   steps:
-  - script: |
-      cd $BUILD_BINARIESDIRECTORY
-      sudo apt -y install cmake gcc g++ ninja-build libncurses5-dev libreadline-dev libssl-dev make zlib1g-dev
-      cmake -G "Ninja" -DCMAKE_BUILD_TYPE=RelWithDebInfo $BUILD_SOURCESDIRECTORY
-      cmake --build .
-      cpack -C Release -G DEB
+  - script: sudo apt-get -y install cmake gcc g++ ninja-build libncurses5-dev libreadline-dev libssl-dev make zlib1g-dev
```

Comment:
I’d stay with `apt`.

Reply:
Just wanted to suppress `WARNING: apt does not have a stable CLI interface. Use with caution in scripts.` in the output

Reply:
Sorry for the noise. Rereading the manual, `apt` seems to be for interactive use, and `apt-get` is the right choice for scripts.

Reply:
No problem.